src/objecttypes/setup_configuration/tests/test_oidc_config.py

```diff
@@ -0,0 +1,73 @@
+from pathlib import Path
+
+from django.test import TestCase
+
+from django_setup_configuration.exceptions import ConfigurationException
+from django_setup_configuration.test_utils import build_step_config_from_sources
+from mozilla_django_oidc_db.models import OpenIDConnectConfig
+from mozilla_django_oidc_db.setup_configuration.steps import AdminOIDCConfigurationStep
+
+DIR_FILES = (Path(__file__).parent / "files/oidc").resolve()
+
+
+class AdminOIDCConfigurationTests(TestCase):
+
+    def setUp(self):
+        OpenIDConnectConfig.clear_cache()
+
+    def test_valid_setup_default(self):
+        config = OpenIDConnectConfig.get_solo()
+        self.assertFalse(config.enabled)
+
+        setup_config = build_step_config_from_sources(
+            AdminOIDCConfigurationStep,
+            str(DIR_FILES / "valid_setup.yaml"),
+        )
+        step = AdminOIDCConfigurationStep()
+        step.execute(setup_config)
```

Comment:
This can be simplified like in the other PR with `execute_single_step` (also applies to test below)